egs/iam/s5/local/chain/run_cnn_chainali_1a.sh

```diff
@@ -33,8 +33,8 @@ alignment_subsampling_factor=1
 chunk_width=340,300,200,100
```

Comment:
Also for this recipe.
Change the description to "chainali_1a is as 1a except it uses chain alignments (using 1a system) instead of gmm alignments" and then append the output (and the command itself) of `compare_wer.sh` for 1a and chainali_1a (after 1 blank line)